tests/services/error-message.service.spec.ts

```diff
@@ -0,0 +1,117 @@
+    });
+  });
+
+  describe('ErrorMessageService: unableToFetchUsersMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToFetchUsersMessage()).toBe('Failed to fetch assignable users for repository');
+    });
+  });
+
+  describe('ErrorMessageService: unableToFetchDataFileMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToFetchDataFileMessage()).toBe('Failed to fetch data file.');
+    });
+  });
+
+  describe('ErrorMessageService: unableToFetchEventsMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToFetchEventsMessage()).toBe('Failed to fetch issue events for repository');
+    });
+  });
+
+  describe('ErrorMessageService: unableToFetchActivityEventsMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToFetchActivityEventsMessage()).toBe('Failed to fetch activity events for repository');
+    });
+  });
+
+  describe('ErrorMessageService: unableToFetchLatestRelease()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToFetchLatestReleaseMessage()).toBe('Failed to fetch latest release.');
+    });
+  });
+
+  describe('ErrorMessageService: unableToFetchSettingsFileMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToFetchSettingsFileMessage()).toBe('Failed to fetch settings file.');
+    });
+  });
+
+  describe('ErrorMessageService: unableToFetchAuthenticatedUsersMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToFetchAuthenticatedUsersMessage()).toBe('Failed to fetch authenticated user.');
+    });
+  });
+
+  describe('ErrorMessageService: unableToOpenInBrowserMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.unableToOpenInBrowserMessage()).toBe('Unable to open this issue in Browser');
+    });
+  });
+
+  describe('ErrorMessageService: applicationVersionOutdatedMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.applicationVersionOutdatedMessage()).toBe('Please update to the latest version of WATcher.');
+    });
+  });
+
+  describe('ErrorMessageService: multipleDropdownOptionsErrorMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.multipleDropdownOptionsErrorMessage()).toBe('Supply either Dropdown option number or text, not both.');
+    });
+  });
+
+  describe('ErrorMessageService: noDropdownOptionsErrorMessage()', () => {
+    it('should return the correct message', () => {
+      expect(ErrorMessageService.noDropdownOptionsErrorMessage()).toBe('No Dropdown identification parameters supplied.');
+    });
+  });
```

Comment:
I'm not sure if there's any benefit to write test cases for the constant provider service. If test cases fail due to future changes in constant values, developers will likely just edit the test cases, resulting in duplicated changes. Also, I think that integration tests for services using these constants should suffice to catch any errors in those constant values.
@CATcher-org/2324s2  what do yall think?